src/Chainweb/BlockHeaderDB/RestAPI.hs

```diff
@@ -408,3 +415,47 @@ headerStreamApi = Proxy
 
 someHeaderStreamApi :: ChainwebVersion -> SomeApi
 someHeaderStreamApi (FromSing (SChainwebVersion :: Sing v)) = SomeApi $ headerStreamApi @v
+
+-- -------------------------------------------------------------------------- --
+-- POW Header
+
+-- | A `BlockHeader` that includes its POW Hash information.
+--
+data PowHeader = PowHeader
+    { _rhHeader :: !(ObjectEncoded BlockHeader)
+    , _rhPowHash :: Text }
+
+instance ToJSON PowHeader where
+    toJSON o = object
+        [ "header"  .= _rhHeader o
+        , "powHash" .= _rhPowHash o ]
+
+instance FromJSON PowHeader where
+    parseJSON = withObject "PowHeader" $ \v -> PowHeader
+        <$> v .: "header"
+        <*> v .: "powHash"
+
+instance ToSchema PowHeader where
+    declareNamedSchema _ = return $ NamedSchema (Just "PowHeader") $ mempty
+        & title ?~ "POW Header"
+        & description ?~ "BlockHeader with its POW Hash"
+        & type_ .~ Just SwaggerObject
+        & required .~ [ "header", "powHash" ]
+
+type PowHeaderApi_
+    = "header"
+    :> Capture "BlockHash" BlockHash
+    :> "pow"
```

Comment:
I think, by making `pow` a resource on its own, one would expect to get only the pow hash without the header here. 

Generally, I don't think that the pow hash deserves to be a resource on its own in the consensus API. This hash is really is just a sequence of random bytes that has no purpose out side pow target validation.